src/fixit/types.py

```diff
@@ -8,9 +8,11 @@
 from typing import Any, Dict, List, Optional
 
 from libcst._add_slots import add_slots
-from libcst.metadata import CodeRange
+from libcst.metadata import CodeRange as CSTCodeRange, CodePosition as CSTCodePosition
```

Comment:
Why the alias? Is this to satisfy the typechecker to re-export? Would __all__ do the same thing?

Reply:
I just wanted to make sure the name `CodeRange` is declared in this file explicitly instead of relying on an import. `__all__` would accomplish the same thing, yeah.